woocommerce-services.php

```diff
@@ -1234,23 +1234,23 @@ public function add_tracking_info_to_emails( $order, $sent_to_admin, $plain_text
 
 			if ( $plain_text ) {
 				echo "\n=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n\n";
-				echo mb_strtoupper( __( 'Tracking', 'woocommerce-services' ), 'UTF-8' ) . "\n\n";
-				echo $markup;
+				echo esc_html(mb_strtoupper( __( 'Tracking', 'woocommerce-services' ), 'UTF-8' )) . "\n\n";
+				echo $markup; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped 
```

Comment:
An alternative to ignoring could be disallowing HTML, eg: echo wp_kses( $raw_response['body'], [] );

Reply:
Thanks for the suggestion. Updated to use `wp_kses` instead.